domains/*.sportlivelk.localplayer.dev.json

```diff
@@ -0,0 +1,16 @@
+{
+    "description": "SPORT LIVE LK",
+    "domain": "localplayer.dev",
+    "subdomain": "*.sportlivelk",
+    "owner": {
+        "repo": "https://github.com/rochana-sadila/register",
+        "email": "mdrochana93@gmail.com"
+    },
+    "record": {
+        "CNAME": "383zyusx94sis.ahost.marscode.site",
```

Comment:
This site could likely fall into the illegal category under piracy laws, which would be against our terms.